src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -323,11 +323,18 @@ private static string[] SplitOnWhiteSpace(string s)
             return s.Split(null);
         }
 
-        private static bool ContainsSpecialCharacters(string element)
+        public static Char[] SpecialAndInvalidCharacters()
         {
-            return element.Contains("*") || element.Contains(".") || element.Contains(" ")
-                || element.Contains("\\");
+            // Excluding white spaces and uncommon characters, only keeping the displayed in the Windows alert
```

Comment:
FYI.. @zeusongit @mjkkirschner see this comment, space is excluded from .NET API

Reply:
Thanks for the clarification @QilongTang 👍🏼